RELEASE-NOTES.md

```diff
@@ -10,144 +10,156 @@ Instead update the vNext section until 4.0.0 is out.
-  - Multiple methods including `gp.prior` now require explicit kwargs.
-- Changes to the BART implementation:
-  - A BART variable can be combined with other random variables. The `inv_link` argument has been removed (see [4914](https://github.com/pymc-devs/pymc3/pull/4914)).
-  - Moved BART to its own module (see [5058](https://github.com/pymc-devs/pymc3/pull/5058)).
-- Changes to the Gaussian Process (GP) submodule (see [5055](https://github.com/pymc-devs/pymc/pull/5055)):
-  - For all implementations, `gp.Latent`, `gp.Marginal` etc., `cov_func` and `mean_func` are required kwargs.
-  - In Windows test conda environment the `mkl` version is fixed to verison 2020.4, and `mkl-service` is fixed to `2.3.0`.  This was required for `gp.MarginalKron` to function properly.
-  - `gp.MvStudentT` uses rotated samples from `StudentT` directly now, instead of sampling from `pm.Chi2` and then from `pm.Normal`.
-  - The "jitter" parameter, or the diagonal noise term added to Gram matrices such that the Cholesky is numerically stable, is now exposed to the user instead of hard-coded.  See the function `gp.util.stabilize`.
-  - The `is_observed` arguement for `gp.Marginal*` implementations has been deprecated.
-  - In the gp.utils file, the `kmeans_inducing_points` function now passes through `kmeans_kwargs` to scipy's k-means function.
-  - The function `replace_with_values` function has been added to `gp.utils`.
-  - `MarginalSparse` has been renamed `MarginalApprox`.
+  - `Model.update_start_values(...)` was removed. Initial values can be set in the `Model.initial_values` dictionary directly.
+  - Test values can no longer be set through `pm.Distribution(testval=...)` and must be assigned manually.
+  - `Transform.forward` and `Transform.backward` signatures changed.
+  - `pm.DensityDist` no longer accepts the `logp` as its first position argument. It is now an optional keyword argument. If you pass a callable as the first positional argument, a `TypeError` will be raised (see [5026](https://github.com/pymc-devs/pymc/pull/5026)).
+  - `pm.DensityDist` now accepts distribution parameters as positional arguments. Passing them as a dictionary in the `observed` keyword argument is no longer supported and will raise an error (see [5026](https://github.com/pymc-devs/pymc/pull/5026)).
+  - The signature of the `logp` and `random` functions that can be passed into a `pm.DensityDist` has been changed (see [5026](https://github.com/pymc-devs/pymc/pull/5026)).
+  - Changes to the BART implementation:
+    - A BART variable can be combined with other random variables. The `inv_link` argument has been removed (see [4914](https://github.com/pymc-devs/pymc3/pull/4914)).
+    - Moved BART to its own module (see [5058](https://github.com/pymc-devs/pymc3/pull/5058)).
+  - Changes to the Gaussian Process (GP) submodule (see [5055](https://github.com/pymc-devs/pymc/pull/5055)):
+    - The `gp.prior(..., shape=...)` kwarg was renamed to `size`.
```

Comment:
@twiecki @michaelosthege Does this have anything to do with the shape/size debate?

Reply:
Yes, it's an example of a multivariate RV where `size` would be the more natural parametrization.

Reply:
should we revert?

Reply:
Yes, this would be quite weird to only advertise `size` for GPs.